source/globalCommands.py

```diff
@@ -4009,6 +4009,18 @@ def _enableScreenCurtain(doEnable: bool = True):
 			else:
 				_enableScreenCurtain()
 
+	@script(
+		description=_(
+			# Translators: Describes a command.
+			"Toggles through paragraph navigation styles",

```

Comment:
For options with more than two states, we usually do not use "toggle".

```suggestion
			"Cycles through paragraph navigation styles",
```